j/Jazz-mmc.md

```diff
@@ -0,0 +1,9 @@
+# Artist name
+
+![alt text](jazz.JPG)
```

Comment:
Try to describe this image in the alt text.
Please also make the image filename the same as the artist's filename.